src/psyclone/psyir/transformations/parallel_loop_trans.py

```diff
@@ -369,6 +368,8 @@ def apply(self, node, options=None, verbose: bool = False,
             else:
                 list_of_names = ignore_dependencies_for
         else:
+            # TODO 2668 - options dict is deprecated.
+            print(self._deprecation_warning)
```

Comment:
`warnings.warn(self._deprecation_warning, DeprecationWarning)` ? if that is what we decided, otherwise a TODO to the logging if this is not fully decided

Reply:
Done, and updated this bit in the dev guide too.

Reply:
The pytest output now shows a bunch of warnings though - do we care about these being in the output of the pytest runs or not? We probably should update these tests to use the new options at some point (though we do need to cover the options section).

Reply:
Yeah, this is a lot of visual noise, can you add `filterwarnings = ignore::DeprecationWarning ` in the setup.cfg [tool:pytest] section

Reply:
Added and I no longer see them.